bubbles/tl_commands.py

```diff
@@ -2,12 +2,8 @@
 import logging
 import sys
 import traceback
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 
-# from bubbles.commands.periodic.activity_checkin import (
```

Comment:
Are we removing the command this goes with or did ruff just nuke it because it's commented out?

Reply:
I removed it manually